src/stories/search-result-element/search-result-element.scss

```diff
@@ -0,0 +1,45 @@
+.search-result-element {
+  padding: 16px 24px;
+  display: grid;
+  grid-template-columns: min-content 1fr;
+  gap: 16px;
+
+  @include breakpoint-s {
+    gap: 24px;
+    padding: 24px;
+    grid-template-columns: min-content max-content 1fr;
+  }
+}
+
+.search-result-element__cover {
+  grid-row: span 2;
+
+  @include breakpoint-s {
+    grid-row: auto;
+  }
+}
+
+.search-result-element__availability {
```

Comment:
About availability - they mention two different cases: 
- one where there is one line of availability buttons:
![image](https://user-images.githubusercontent.com/28546954/171380158-9148d590-ba14-478b-b565-05808b17fbb1.png)

- and one where there are two:
![image](https://user-images.githubusercontent.com/28546954/171380007-21b925b2-77ec-4b7b-a152-3f16950583d6.png)

Also notice how they clearly all have the same alignment to left starting at 90px to the right of the headline:
![image](https://user-images.githubusercontent.com/28546954/171380462-9c99d911-4cb2-4ed4-bf1b-479264f83ab1.png)